src/components/BlogPost.stories.js

```diff
@@ -9,17 +10,31 @@ export default {
   excludeStories: /.*Data$/,
 };
 
+export const actionsData = {
+    onLikedPost: action('like-post'),
+  };
+
 const Template = (args, { argTypes }) => ({
   components: { BlogPost },
   props: Object.keys(argTypes),
-  template: '<BlogPost v-bind="$props" />',
+  methods: actionsData,
+  template: '<BlogPost v-bind="$props" @liked-post="onLikedPost/>',
```

Comment:
@ricomadeit This might be the issue: missing closing `"`

Reply:
**More specifically `@liked-post="onLikedPost` needs to be `@liked-post="onLikedPost"`